src/main/java/step2/StringAddCalculator.java

```diff
@@ -0,0 +1,48 @@
+package step2;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    private static final String CUSTOM_DELIMITER = "//(.)\n(.*)";
+    private static final String COMMA_AND_COLON = "[,:]";
+    private static final int GROUP_ONE = 1;
+    private static final int GROUP_TWO = 2;
+
+    public static int splitAndSum(String text) {
+        if (isNullOrBlank(text)) return 0;
```

Comment:
```suggestion
        if (isNullOrBlank(text)) {
            return 0;
        }
```

큰 내용 아니어도 가급적 중괄호는 꼭 붙여주세요.
[핵데이 컨벤션](https://naver.github.io/hackday-conventions-java/#need-braces) 이지만 일반적으로 통용되고 있습니다.